src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -596,6 +645,27 @@ export default class StripViewModel extends BaseModel {
         return SELECTORS.CLASSNAMES.ARRIVAL;
     }
 
+    /**
+     * Return the assigned runway based on whether an aircraft is a `departure` or an `arrival`
+     *
+     * @for AircraftStripView
+     * @method _buildRunway
+     * @return {string}
+     */
+    _buildRunway(aircraftModel) {
```

Comment:
Why not something like the following:

```js
_buildRunway(aircraftModel) {
    let runwayName = '';
    
    if (aircraftModel.isDeparture()) {
        runwayName = aircraftModel.fms.departureRunwayModel.name;
    } else if (aircraftModel.isArrival()) {
        runwayName = aircraftModel.fms.arrivalRunwayModel.name;
    }
          
    return runwayName;
}
```

additionally, loose equal comparisons should throw a linting error: `!=` or `==`, we stick to strict always.

Reply:
Loose equal comparison was my Java autopilot 🙈 but my eslint did not complain about it. But then again, I am not sure that my eslint is working properly.